packages/facility-server/app/routes/apiv1/surveyResponse.js

```diff
@@ -141,3 +145,45 @@ surveyResponse.put(
     res.send(responseRecord);
   }),
 );
+
+surveyResponse.get(
+  '/patient-data-field-association-data/:column',
+  asyncHandler(async (req, res) => {
+    const { models, params, query } = req;
+    const value = query.value;
+    const column = params.column;
+
+    req.checkPermission('read', 'Patient');
+
+    if (!column) {
+      throw new InvalidParameterError('Column parameter is required');
+    }
+    if (!value) {
+      res.json({
+        data: null,
+      });
+      return;
+    }
+
+    const [modelName, fieldName] = PATIENT_DATA_FIELD_LOCATIONS[column] || [];
```

Comment:
we should probably return an error if the column is not available to be queried (ie not in the constant) rather than returning null